changelog/12460.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
```

Comment:
We can remove the changelog entry from this PR. We'll submit a single entry for the entirety of the feature once it's complete.